docs/guide/database-access.md

```diff
@@ -38,22 +38,18 @@ There are references to other core apostrophe collections that can be accessed i
 
-Keep in mind, this update to the record happened *after* the requested piece was already returned from the database because we used the `beforeShow()` method. So, if five people had looked at that blog article previously, `data.piece.views` in the template would be equal to 5, but the database record views would now be 6 due to our `$inc`.
+Keep in mind, this update to the record happened *after* the requested piece was already returned from the database because we used the `beforeShow()` method. So, if five people had looked at that blog article previously, `data.piece.views` in the template would be equal to 5 for the duration of the current request, but the database record views would now be 6 due to our `$inc`.
 
 In addition to the `$inc` operator, there are a number of other MongoDB operators where it makes sense to access database directly. This includes `$set`, `$pull`, `$push`, `$addToSet`, and `$unset`. You can read more about their usage in the MongoDB [documentation](https://www.mongodb.com/docs/v6.0/reference/operator/update/).
 
-## Modifying collections using tasks
-
-Let's look at another typical use case for directly modifying our collections. Any module can add command line (CLI) tasks through the `tasks(self)` configuration property.
-
-The Apostrophe modules expose multiple tasks, for example, adding users through `node app @apostrophecms/user:add <name> <role>`, or completely resetting your database (dangerous!) through `node app @apostrophecms/db:reset`. The [`task()` function](https://v3.docs.apostrophecms.org/reference/module-api/module-overview.html#middleware-self) takes the as an argument, returning one or more CLI-invokable functions. You can access and modify documents within a CLI function just as you would from any module method.
+These same methods of altering documents outside of the Apostrophe model layer can be used in [tasks](/reference/module-api/module-overview.html#tasks-self) created by any module. As examples, the Apostrophe modules expose multiple tasks like adding users through `node app @apostrophecms/user:add <name> <role>` from the `@apostrophecms/user` module or completely resetting your database (dangerous!) through `node app @apostrophecms/db:reset`.
 
 ## Making your own database connections
 
 While the core of apostrophe depends on connections to MongoDB through the mongodb native module, that doesn't mean that your custom modules are limited in the same way. If you feel like using Mongoose, go ahead! If you want to use a different database, why not!
 
-Just install the packages you need and invoke your database connections with `await` within the `async init(self)` function of your module.
\ No newline at end of file
+Just install the packages you need and invoke the functions of the appropriate library with `await` within the `async init(self)` function of your module.
```

Comment:
Then attach the returned connection object to a property of `self`, granting access to it later in any method, handler, route, etc. of the module. You should also clean up such connections in an `apostrophe:destroy` handler:

```
handlers(self) {
   return {
      'apostrophe:destroy': {
        async closeMyConnection() {
          await self.myConnection.close();
        }
      }
   };
}
```

Reply:
Added